src/jdk.hotspot.agent/macosx/native/libsaproc/libproc_impl.h

```diff
@@ -42,6 +42,7 @@
 #define register_t uint64_t
 #endif
 
+#if defined(__x86_64__)
```

Comment:
can we make it uniformly with MacosxDebugfferLocal.m here please?

using amd64 define instead of __x86_64__

arm64 instead of __arm64__